src/components/Form/Step3/HowProcess/HowProcess.js

```diff
@@ -1,11 +1,13 @@
 import React, { useContext } from 'react';
 // Import contexts
-import { FormContext } from 'globalState/FormContext';
+import { FormDataContext } from 'globalState/FormDataContext';
+import { useFormContext } from 'react-hook-form';
 // Import components
 import Radios from 'components/shared/FormElements/Radios/Radios';
 
 const HowProcess = () => {
-  const [, formDispatch] = useContext(FormContext); // Get the state of form data from FormContext
+  const [, formDispatch] = useContext(FormDataContext); // Get the state of form data from FormDataContext
+  const { register } = useFormContext();
 
   // Update customerType on radio button change
   const handleRadioChange = (e) =>
```

Comment:
I've noticed this handleRadioChange is used in a few places...could go in a custom hook.
Not essential but maybe a future update to save going in 10 different files to change the same logic.